src/fprime_extras/devupdate.py

```diff
@@ -47,7 +52,7 @@ def check_version(sha, branch='main'):
     gh_sha = get_github_version(branch=branch)
     # print(gh_sha)
     # print(gh_sha[0:len(sha)])
-    return sha != gh_sha[0:len(sha)]
+    return sha != gh_sha[0:len(sha)] and gh_sha != ''
```

Comment:
This will silently ignore any issues with the github sha. @SterlingPeet, if you want there to be an error message like you mentioned in #3 then I may need to change this

Reply:
I definitely don't want a silent ignore, not for people using a dev branch that is ahead of a release.  More logic is needed to support that, but for now it would be best to emit an error message and continue working (as mentioned in #3).

Reply:
I fixed the exception check. This will now emit the warning message on devupdate.py:66, but still continue execution

Reply:
does this fix #3 altogether?

Reply:
Yes, now any error with the github hash will not cause fprime-extras to fail.

Reply:
sweet, in that case, add the phrase ``Closes #3`` to the appropriate git commit message.

Reply:
Actually, I think I'm gonna just pull it in and i'll tweak the message myself